packages/assets-controllers/src/NftController.test.ts

```diff
@@ -4378,4 +4394,25 @@ describe('NftController', () => {
 
     expect(updateNftMetadataSpy).not.toHaveBeenCalled();
   });
+
+  it('fetchNftCollectionsMetadata fetches NFT collections metadata successfully', async () => {
```

Comment:
What are your thoughts on wrapping this test in a `describe` block named after the method you want to test? For instance:

``` typescript
  describe('fetchNftCollectionsMetadata', () => {
    it('fetches NFT collections metadata successfully', async () => {
      // ...
    });
  });
```

I realize that this is not the style for other tests in this file, but it's a principle that we've been trying to move to as a whole. More about  this here: https://github.com/MetaMask/contributor-docs/blob/main/docs/unit-testing.md#use-describe-to-group-tests-for-the-same-functionmethod

Reply:
I like the way of putting multiple `describe` blocks as well, the only reason that I didn't here was that this was the only test we run for the method.
But if we are happy to have one test inside describe block, here it is https://github.com/MetaMask/core/pull/4506/commits/24ac62f90f7e859f1810a8a7ba276ce9ef69290f